### PR TITLE
SC-1190 

### PR DESCRIPTION
**Pull request by anmol2302**
<b>Ticket Ref:</b> [SC-1190](https://project-sunbird.atlassian.net/browse/SC-1190)
1: Added validation and JsonKeys and Actor Operation for request assignKeys
2: This class will validate the mandatory params id, signKeys, encKeys
3: If anyone is missing it will through mandatory params missing
4: apart from that its also checking the size of signKeys, encKeys, and their object type should be List.

### CODE REVIEW

File: sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.sunbird.common.responsecode.ResponseCode;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+/**
+ * this class is used to validate the request of the OrgAssignKeys Controller
+ * @author anmolgupta
+ */
+public class AssignKeyValidator extends BaseRequestValidator {
```
Comment (manzarul) [inline sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java]: AssignKeyValidator to keyManagementValidator

File: sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            throw new ProjectCommonException(
+                    ResponseCode.mandatoryParamsMissing.getErrorCode(),
+                    ResponseCode.mandatoryParamsMissing.getErrorMessage(),
+                    ResponseCode.CLIENT_ERROR.getResponseCode(),key);
+        }
+    }
+
+    private void validateSize(String key){
+        if(((List)request.get(key)).size()==0){
+            throw new ProjectCommonException(
```
Comment (manzarul) [inline sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java]: add message as key can't be empty.

Comment (anmol2302) [reply sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java]: Acknowledged and Executed

File: sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java (hunk 1 above)
Comment (anmol2302) [reply sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java]: Acknowledged and Executed

**Pull request merged**